common/next/next.config.js

```diff
@@ -85,6 +85,7 @@ module.exports = function (webpack) {
 
   return withMDX(
     withTM({
+      webpack5: false,
```

Comment:
How comes?

Reply:
Because we were sharing config, this was required because the shape of the configuration object has changed between webpack 4 and 5. Now that we're splitting out the configs, this definitely doesn't need to be here. And we can look at what it would take to get it working with webpack 5 in the identity app